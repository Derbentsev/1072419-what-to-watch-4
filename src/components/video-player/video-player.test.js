import VideoPlayer from './video-player';


const mocks = {
  src: `src1`,
  poster: `poster1`
};

it(`Render Video Player`, () => {
  const tree = renderer
    .create(<VideoPlayer
      src = {mocks.src}
      poster = {mocks.poster}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
