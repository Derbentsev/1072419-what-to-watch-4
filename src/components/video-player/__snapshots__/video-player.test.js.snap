// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Video Player 1`] = `
<div
  className="small-movie-card__poster"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <video />
</div>
`;
